src/components/select/simple-select/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './simple-select.component';
```

Comment:
We're exposing two ways of using Select.

- `import {Select} from 'carbon-react/lib/components/select`
- `import Select from 'carbon-react/lib/components/select/simple-select`

We should have only one way to do this.

Reply:
Unresolved.

Reply:
removed index file